### PR TITLE
open-cluster-management/backlog#13792 namespace

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by xiangjingli — APPROVED** [review]

**Review by fxiang1 — APPROVED** [review]
LGTM!

File: install/requirements.adoc [hunk 1]
```diff
@@ -131,4 +131,49 @@ In general, the application deployment communication is one way from a managed c
...
+   
+    On the managed cluster, there is only one application lifecycle pod that is installed in the `open-cluster-management-agent-addon` namespace:
+    ** klusterlet-addon-appmgr
++
+    The namespace "open-cluster-management-agent-addon" needs to access the Kube API on port 6443
+    
+* Governance and risk
++
+The namespace `open-cluster-management` needs to access the following namespaces:
+** `openshift-kube-apiserver` on port TCP/6443
```
Comment (gparvin) [inline install/requirements.adoc]: For grc it looks like you have the content from option 2 here in option 1.  Can you use the same wording as you have for ALC:
- The namespace `open-cluster-management` needs to access the Kube API on port 6443.
- The namespace `open-cluster-management` needs to access the OpenShift DNS on port 5353.
- On the managed cluster, the namespace "open-cluster-management-agent-addon" needs to access the Kube API on port 6443

File: install/requirements.adoc [hunk 2]
```diff
@@ -131,4 +131,49 @@ In general, the application deployment communication is one way from a managed c
...
+| Cluster | Area | Namespace | Target | Port
+
+| Hub | Application lifecycle | open-cluster-management | Console API | 4000
+| Hub | Application lifecycle | open-cluster-management | Application UI | 3001
+| Hub | Application lifecycle (backend) | open-cluster-management- | Kube API | 6443
+| Managed | Application lifecycle (backend) | open-cluster-management-addon | Kube API | 6443
+| Hub | Governance and risk | open-cluster-management | openshift kube-apiserver | 6443/TCP
+| Hub | Governance and risk | open-cluster-management | openshift-dns | 5353/UDP
+
+See the https://access.redhat.com/articles/6218901[{product-title} {product-version} Support matrix] for additional information.
```
Comment (gparvin) [inline install/requirements.adoc]: When I viewed these changes in github, the additional information link was showing up in the last box of the option 2 table.  If you go with Option 2, can you double check the additional information link doesn't get stuck in the table?

**Review by gparvin — COMMENTED** [review]
To me option 2 seems nicer since all the info is in a table.  I like getting the details with less to read I guess.

**Comment by cadawson** [comment]
I probably should have guessed that there would be different opinions on it  ;-). Since 2 of you preferred option 1, and Gus provided input for me to work with option 1, I went with that one. Please give it a final review before I merge. Also, I plan to backport this to 2.3. If it is not the same in 2.3, let me know what differences exist.
@bjoydeep : Do you have any additions from Observability? 
Thank you! @fxiang1 @xiangjingli @gparvin

**Review by xiangjingli — APPROVED** [review]

**Comment by xiangjingli** [comment]
Thanks @cadawson! This looks good to me

**Review by gparvin — APPROVED** [review]

**Pull request merged**